.github/workflows/rust.yml

```diff
@@ -165,11 +189,44 @@ jobs:
         run: |
           git diff --staged --no-ext-diff --quiet --exit-code
 
+  publish-release:
+    name: Check version and publish release
+    runs-on: ubuntu-latest
+    needs:
+      ["linux", "mac", "windows"]
+    if: ${{ !contains(needs.*.result, 'failure') && !contains(needs.*.result, 'cancelled') }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/download-artifact@v4
+        with:
+          merge-multiple: true
+      - id: check-tag
+        if: ${{ github.ref == 'refs/heads/main' && github.event_name == 'push' }}
+        run: |
+          RELEASE_TAG=mozjs-sys-v$(cargo metadata --format-version=1 --no-deps | jq -r '.packages[] | select(.name == "mozjs_sys") | .version')
+          if git show-ref --tags --verify --quiet "refs/tags/${RELEASE_TAG}"; then
+            RELEASE_TAG=''
+          fi 
```

Comment:
Or did you refer to that code? In that case we can simply remove this and make next step to not fail, which will ensure to test all merged commits with latest release available.

Reply:
Oh sorry I didn't read your comments before pushing. `gh create release ... || exit 0` seems simpler. I'll update it.

Reply:
What you did is actually way better!

Reply:
ah okay! Let me revert it. 😅